src/commands/compile/compileCommand.ts

```diff
@@ -62,18 +62,24 @@ export class CompileCommand extends TargetCommand {
     sourcefilePathParts.splice(-1, 1)
     const sourceFolderPath = sourcefilePathParts.join(path.sep)
     const leafFolderName = sourceFolderPath.split(path.sep).pop() as string
-    return value
-      ? path.isAbsolute(value)
+
+    let outputPath: string
+    if (value)
```

Comment:
we've agreed to wrap such statements in `{}`

Reply:
Refactored this piece of code, and mentioned in above [discussion](https://github.com/sasjs/cli/pull/908#discussion_r690070703).